consensus/core/src/constants.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::Round;
+
+/// Default wave length for all committers. A longer wave length increases the
+/// chance of committing the leader under asynchrony at the cost of latency in
+/// the common case.
+pub const DEFAULT_WAVE_LENGTH: Round = MINIMUM_WAVE_LENGTH;
+
+/// We need at least one leader round, one voting round, and one decision round.
+pub const MINIMUM_WAVE_LENGTH: Round = 3;
```

Comment:
maybe those can be `pub(crate)` instead?

Reply:
If we want to keep the `constants.rs` then let's all agree that this is the place where we'll put all our constants , otherwise we'll end up a quite partitioned logic